docs/specification/README.md

```diff
@@ -244,8 +244,8 @@ Attachment records must not appear within a chunk.
 | Bytes | Name | Type | Description |
 | --- | --- | --- | --- |
 | 4 + N | name | String | Name of the attachment, e.g "scene1.jpg". |
-| 8 | created_at | Timestamp | Time at which the attachment was created. |
 | 8 | log_time | Timestamp | Time at which the attachment was recorded. |
+| 8 | creation_time | Timestamp | Time at which the attachment was created. If not available, must be set to zero. |
```

Comment:
I think `create_time` might be more in line with the tense used elsewhere? (`log_time`, `publish_time`, `start_time`, `end_time`)